test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -476,11 +476,11 @@ public void testIncludeAllColumnsAsMetadata_mcSet() throws Exception {
     final Config config = createStandardConfig(moreEntries);
     DatabaseAdaptor adaptor = new DatabaseAdaptor();
     adaptor.init(TestHelper.createConfigAdaptorContext(config));
-    assertEquals("MetadataColumns({col2=gsa2, db_col1=gsa1})",
+    assertEquals("{col2=gsa2, db_col1=gsa1}",
```

Comment:
Not nearly as much of this as a I thought. Since we're not using Guava, what do you think about using something like this:

  @SuppressWarnings("unchecked")
  private <T> Map<T, T> asMap(T... entries) {
    if (entries.length % 2 != 0) {
      throw new IllegalArgumentException(
          "A map requires an even number of entries");
    }
    Map<T, T> map = new HashMap<T, T>();
    for (int i = 0; i < entries.length - 1; i += 2) {
      map.put(entries[i], entries[i + 1]);
    }
    return map;
  }